packages/flutter_test/lib/src/controller.dart

```diff
@@ -530,9 +535,25 @@ abstract class WidgetController {
   }) {
     assert(kDragSlopDefault > kTouchSlop);
     return TestAsyncUtils.guard<void>(() async {
-      final TestGesture gesture = await startGesture(startLocation, pointer: pointer, buttons: buttons);
+      final TestGesture gesture = await startGesture(
```

Comment:
We can restore all the changes here as `_separateDragOffset` is not going to be used by anyone else

Reply:
Done.